G Suite migration documentation update (#2118)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md

4. On the **Service account permissions (optional)** screen, click **Continue**.

5. On the **Grant users access to this service account (optional)** screen, click **Done**.

> 6. Once you have been returned to the page listing the service accounts, click on the **Email** for the Service Account you just created to enter the details page, then click the **Edit** button. Alternately, you can click the ellipsis under the **Actions** column and select the **Edit** action.

Reviewer: ```suggestion
6. Once you have been returned to the page listing the service accounts, click on the **Email** for the Service Account you just created to enter the details page, then click the **Edit** button. Alternatively, you can click the ellipsis under the **Actions** column and select the **Edit** action.
```

7. On the Service account details page, note the **Unique ID**. This is the ClientId that you will provide later in the instructions for [Grant access to the service account for your Google tenant](#grant-access-to-the-service-account-for-your-google-tenant).

   ![Unique ID](../media/gsuite-mig-6-unique-id.png)

> 8. If you see an area that says **Show Domain-Wide Delegation**, click to expand that section. Then, enable the checkbox for **Enable G Suite Domain-wide Delegation**, and then click **Save** at the bottom of the page.

Reviewer: ```suggestion
8. If you see an area that says **Show Domain-Wide Delegation**, click to expand that section. Then, enable the checkbox for **Enable G Suite Domain-wide Delegation** and click **Save** at the bottom of the page.
```

   ![Domain-Wide Delegation](../media/gsuite-mig-7-enable-domain-delegation.png)

9. On the Service account details page, click **Add Key** and select the **Create new key** option.

   ![Create key](../media/gsuite-mig-3-createkey.png)

10. Under **Key type**, make sure **JSON** is selected, and then click **Create**.

    ![JSON key](../media/gsuite-mig-4-json.png)

> 11. Keep track of the JSON keyfile that is automatically downloaded, as you will need its filename during the steps under [Create a migration endpoint in Microsoft 365 or Office 365](#create-a-migration-endpoint-in-microsoft-365-or-office-365). Close the pop-up dialog and click **Save**.

Reviewer: ```suggestion
11. Keep track of the JSON key file that is automatically downloaded, as you will need its filename during the steps under [Create a migration endpoint in Microsoft 365 or Office 365](#create-a-migration-endpoint-in-microsoft-365-or-office-365). Close the pop-up dialog and click **Save**.
```

1. Go to the [G Suite Admin page](https://admin.google.com/AdminHome) and sign in as a G Suite admin for your tenant.

> 2. Click **Domains**, and then **Manage domains**, and then click **Add a domain**.

Reviewer: ```suggestion
2. Click **Domains**, then **Manage domains**, and then click **Add a domain**.
```

   ![Add domain](../media/gsuite-mig-9-sub-domain-O365.png)

4. Follow any subsequent steps that are then required to verify your domain, making sure that the status is shown as **Active**. Note that if you chose a subdomain of your primary domain in step 3 above, your new domain may have been verified automatically.

   ![Verify domain](../media/gsuite-mig-10-verify-domain.png)

5. Log into your DNS provider and update your DNS records so that you have an MX record at the domain you created above in step 3, pointing to Microsoft 365 or Office 365. Ensure that this domain that you created above is an accepted domain in Microsoft 365 or Office 365. Follow the instructions in [Add a domain to Microsoft 365](https://docs.microsoft.com/microsoft-365/admin/setup/add-domain) to add the Microsoft 365 or Office 365 routing domain ("o365.fabrikaminc.net") to your organization and to configure DNS to route mail to Microsoft 365 or Office 365.

## Create a sub-domain for mail routing to your G Suite domain

1. Go to the [G Suite Admin page](https://admin.google.com/AdminHome) and sign in as a G Suite admin for your tenant.

> 2. Click **Domains**, and then **Manage domains**, and then click **Add a domain alias**.

Reviewer: ```suggestion
2. Click **Domains**, then **Manage domains**, and then click **Add a domain alias**.
```

Before proceeding with either method, make sure that Mail Users have been provisioned for every user in the organization who will be migrated (either now or eventually). If any users aren't provisioned, provision them using the instructions in [Manage mail users](https://docs.microsoft.com/exchange/recipients-in-exchange-online/manage-mail-users).

> You can also deploy Azure Active Directory (Azure AD) Connect to provision your Mail Users, see [Deploy Microsoft 365 Directory Synchronization in Microsoft Azure](https://docs.microsoft.com/office365/enterprise/deploy-office-365-directory-synchronization-dirsync-in-microsoft-azure) for an overview, and [Set up directory synchronization for Microsoft 365](https://docs.microsoft.com/office365/enterprise/set-up-directory-synchronization) for setup instructions. Then, you need to deploy an Exchange server in your on-premises environment for user management, and mail-enable your users using this server. For more information, see [How and when to decommission your on-premises Exchange servers in a hybrid deployment](https://docs.microsoft.com/exchange/decommission-on-premises-exchange) and [Manage mail users](/Exchange/ExchangeServer/recipients/mail-users.md). Once the Mail Users have been created in Microsoft 365, the Azure AD Connect will need to be disabled in order to allow the Migration process to convert these users into Mailboxes - see [Turn off Directory Synchronization](https://docs.microsoft.com/en-us/office365/enterprise/turn-off-directory-synchronization).

Reviewer: ```suggestion
You can also deploy Azure Active Directory (Azure AD) Connect to provision your Mail Users, see [Deploy Microsoft 365 Directory Synchronization in Microsoft Azure](https://docs.microsoft.com/office365/enterprise/deploy-office-365-directory-synchronization-dirsync-in-microsoft-azure) for an overview, and [Set up directory synchronization for Microsoft 365](https://docs.microsoft.com/office365/enterprise/set-up-directory-synchronization) for setup instructions. Then, you need to deploy an Exchange server in your on-premises environment for user management, and mail-enable your users using this server. For more information, see [How and when to decommission your on-premises Exchange servers in a hybrid deployment](https://docs.microsoft.com/exchange/decommission-on-premises-exchange) and [Manage mail users](/Exchange/ExchangeServer/recipients/mail-users.md). Once the Mail Users have been created in Microsoft 365, the Azure AD Connect will need to be disabled in order to allow the migration process to convert these users into mailboxes - see [Turn off directory synchronization for Microsoft 365](https://docs.microsoft.com/office365/enterprise/turn-off-directory-synchronization).
```

Reviewer: Please edit the last sentence as indicated (note capitalization) and remove the locale from the link.
